src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,695 @@
+
+<Callout variant="tip">
+If you need to view a bearer token later, the only way to do this is to generate a new one, and that will invalidate the old one and any integrations using the old token.
+</Callout>
+
+## Create users and user groups in your system [#create-in-idp]
+
+The SCIM API is typically used by scripts for importing users and groups into New Relic from a database or a third-party identity provider that doesn't have pre-configured configs for New Relic. 
+
+If you want to use the SCIM API  custom application or for ad-hoc requests, proceed to learn how to connect to the SCIM API.
+
+## Connect to the SCIM API [#connect]
+
+The SCIM API is available at `https://scim-provisioning.service.newrelic.com/scim/v2` and this URL is viewable in the authentication domain settings page. To access the SCIM API, your client must include a **bearer token** with each request. The token is displayed after saving your Authentication Domain configuration.
+
+If you're using a third-party identity provider, configure it to use **Bearer token authorization** and plug in your API access token. Refer to your identity provider's documentation for help configuring this. Once configured, you're all set to import users and groups. 
+
+Rather than reading the entire SCIM protocol RFCs, there are three specific sections you may find valuable: See [RFC 7643 - SCIM Core Resources and Extensions](https://datatracker.ietf.org/doc/html/rfc7643#section-4) and [RFC 7643 - JSON Representation](https://datatracker.ietf.org/doc/html/rfc7643#section-8) for the specifics. Refer to [RFC 7644 - SCIM Protocol](https://datatracker.ietf.org/doc/html/rfc7644) for more information about the protocol used in this tutorial.
+
+For all requests to the SCIM API, you must provide the bearer token in an `Authorization` header. Here's an example with `curl`:
+
+```
+curl -X 'GET' -H 'Accept: application/json' -H "Authorization: Bearer $TOKEN" 'https://scim-provisioning.service.newrelic.com/scim/v2/Users'
+```
+
+Any request in the rest of this tutorial will receive a **401 Unauthorized** response if the API access token is missing or invalid.
+
+Example response:
+
+<Collapser title="401 Unauthorized">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:api:messages:2.0:Error"
+    ],
+    "detail": "invalid authentication token",
+    "status": "401"
+  }
+  ```
+</Collapser>
+
+## Create users in your authentication domain [#create-users]
+
+You can use the SCIM API to send a `POST` request to `/scim/v2/Users` to create a user. The following user attributes are **required**:
+* `userName` This identifier **must** be unique within an authentication domain. Use the user's email address. 
+* `emails` The single email address of the user (despite it being called `emails`, only one is accepted).
```

Comment:
Should someone use the same email they used in userName? Might be good to make this clear.